homeassistant/components/v2c/sensor.py

```diff
@@ -100,6 +116,31 @@ def get_meter_value(value: SlaveCommunicationState) -> str:
         value_fn=lambda evse_data: evse_data.battery_power,
         entity_registry_enabled_default=False,
     ),
+    V2CSensorEntityDescription(
+        key="ssid",
+        translation_key="ssid",
+        icon="mdi:wifi",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda evse_data: evse_data.SSID,
+        entity_registry_enabled_default=False,
+    ),
+    V2CSensorEntityDescription(
+        key="ip_address",
+        translation_key="ip_address",
+        icon="mdi:ip",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda evse_data: evse_data.IP,
+        entity_registry_enabled_default=False,
+    ),
+    V2CSensorEntityDescription(
+        key="signal_status",
```

Comment:
Does this sensor return a value that would fall under the `SensorDeviceClass.SIGNAL_STRENGTH` device class?

Reply:
Or if it's returning names "good", "bad", let's make this an enum sensor

Reply:
it returns an integer which certainly is not in dBm ... it's probably a good/bad sensor but I don't know the range...